wasm/src/lib.rs

```diff
@@ -495,6 +495,21 @@ pub async fn capacity_limit() -> Result<u64, JsError> {
         .await?)
 }
 
+#[wasm_bindgen(js_name = feeInfo)]
+pub async fn fee_info() -> Result<FeeInfo, JsError> {
+    Ok(app_client()
+        .query(|app: InnerApp| {
+            let building = app.bitcoin.checkpoints.building()?;
+            let est_miner_fee =
+                building.fee_rate * app.bitcoin.checkpoints.active_sigset()?.est_witness_vsize();
```

Comment:
This should be multiplied by `app.bitcoin.checkpoints.config.user_fee_factor` then divided by 10,000 (since the parameter is denominated in basis points), see here: https://github.com/nomic-io/nomic/commit/779b5c0f7330871ba5c9602bc34873e926e7e061